tests/lib/assert.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  _assert (value, msg) {
+    if (!value) {
+      throw new Error(msg || `AssertionError: when evaluation ${value}`)
+    }
+  },
+  equal (value1, value2, msg) {
```

Comment:
suggestion: add linebreaks in between function definitions. it makes it easier to scan and read.